en/docs/modules/misc/VideoChip.md

```diff
@@ -120,6 +126,53 @@ Draws a portion of the spritesheet, `SpriteSheet` (defined by `spriteOffset` and
 Draws a render buffer (supposedly coming from **Webcam** component) at the desired position, width and height.
 
 
+## `PixelData`
+⚠️ **This feature is new, as of Retro Gadgets 0.1.5, and not fully tested yet.**  
+<!-- I don't know if this is the best description for it -->
+PixelData is a sort of buffer you can write to, and later display.
+It's a faster way to draw indivdual pixels to the screen compared to `SetPixel` on the VideoChip.
+
+### new(width `number`, height `number`, color `color`)
+Constructor used to create PixelData. `width` and `height` represent the size of it, while `color` defines the color with which it is initialized.
+
+### After creating a PixelData you can use methods below to manipulate it.
+
+### Clear(color `color`)
+Clears the PixelData to a specified color.
+
+### SetPixel(x `number`, y `number`, color `color`) --> `color`
```

Comment:
The current style omits any sort of return type symbology, based of the style used by the in-game multitool.
```suggestion
### SetPixel(x `number`, y `number`, color `color`) `color`
```
Think we should have `-->`? Open an issue to discuss with the community!